src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,181 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Importing classes under `duke.command`  one by one will make it easier to identify and improves readability 😄

Reply:
Yes I believe prof has cautioned against the use of wildcard imports.